config/azure/build.yaml

```diff
@@ -80,24 +87,35 @@ jobs:
    - script: |
        echo BUILD_TYPE=${{ parameters.buildType }}
        g++ --version
-       scons -j ${{ parameters.threads }} package ENV_VARS_TO_IMPORT=PATH BUILD_TYPE=${{ parameters.buildType }} DELIGHT_ROOT=$DELIGHT ARNOLD_ROOT=$ARNOLD_ROOT BUILD_DIR=./build INSTALL_DIR=./install/$(Gaffer.Build.Name) BUILD_CACHEDIR=sconsCache
+       rm -rf .sconsign.dblite .sconf_temp
```

Comment:
Do we need this or is it left over from testing? I wouldn't expect either of these files to exist at this point.

Reply:
Totally, removed [here](https://github.com/GafferHQ/gaffer/pull/3877/commits/f6654a6b2b7706c799452e19b162104bc21729f5)